clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -145,6 +145,7 @@
 // CHECK-NOT: __riscv_zacas {{.*$}}
 // CHECK-NOT: __riscv_zalrsc {{.*$}}
 // CHECK-NOT: __riscv_zcmop {{.*$}}
+// CHECK-NOT: __riscv_zalasr {{.*$}}
```

Comment:
this list is roughly in alphabetical order, can you alphabetize zalasr above zalrsc

Reply:
Done, there was another place in the same file I fixed